spec/integration/worker_async_spec.rb

```diff
@@ -81,24 +81,21 @@ module Backend
         # verify that the stats usage keys have been updated correctly.
 
         n_reports = 10
-        work_task, stats_key = nil
 
-        Timecop.freeze(current_time) do
-          without_resque_spec do
-            multi_job_adder.call(n_reports) # add jobs before worker started
-          end
+        without_resque_spec do
+          multi_job_adder.call(n_reports) # add jobs before worker started
+        end
 
-          work_task = Async { subject.work }
+        work_task = Async { subject.work }
 
-          stats_key = Stats::Keys.application_usage_value_key(
-            service_id, app_id, metric_id, Period[:day].new(current_time)
-          )
+        stats_key = Stats::Keys.application_usage_value_key(
+          service_id, app_id, metric_id, Period[:day].new(current_time)
```

Comment:
So this should work as long as don't run the tests precisely at midnight so that some records go to last day and some to next day?

Maybe good enough, but still why remove timecop?

Reply:
I'm my tests locally, `:current_time` is evaluated only once when running

```ruby
let(:current_time) { Time.now }
```

Then it keeps the same value during all the test execution

Reply:
`current_time` will obviously stay the same. But we are adding entries which don't use `current_time` but the actual time they are created. My assumption is that's the reason for timecop usage originally.

Reply:
Right, I'll add it back.